docs/application/dotnet/samples/general/overview.md

```diff
@@ -40,8 +40,8 @@ for TD:
 			<td><img alt="" height="180" src="media/wplayingwithhwInputs.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/PlayingWithHWInputs" target="_blank"><strong>(W) PlayingWithHWInputs</strong></a></p>
-			<p>This sample application demonstrates how to handle rotary events from bezel of a wearable device and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>. In addition, it shows the way how to detect when the back button is pressed.</p>
-			<p>This application uses Tizen.Wearable.CircularUI API.<br>
+			<p>This sample application demonstrates how to handle rotary events from bezel of a wearable device and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>. In addition, it shows the way how to detect when the back button is pressed.</p>
```

Comment:
Gear device has the rotating bezel. This sample is using this feature and try to explain how to use it.

Reply:
Suggest rephrasing to:

This sample application demonstrates how to handle rotary events of the bezel of a wearable device and how to create circular UI using....

Please provide more information for the following sentence:
In addition, it shows the way how to detect when the back button is pressed.

Can we rephrase it to the following and does it convey the same meaning:
In addition, it shows how to detect the navigation path when the back button is pressed.

Reply:
Hello, please clarify this doubt.

Reply:
@sgchoi5,
Thank you for the response.
Please let me know if it would it be okay to rephrase the last sentence as follows:

In addition, it demonstrates how to use the rotating bezel.